### PR TITLE
Add k8s 1.22 makefile e2e test

### PR DESCRIPTION
**Pull request by daniel-spray**
#### What does this PR do?
Adds the 1.22 e2e test to the Makefile and adds more info on test logs.
#### Where should the reviewer start?
MakeFile
#### How should this be manually tested?
Run `make test-e2e-1.22` and `make test-e2e-all`
#### Any background context you want to provide?
Part of 1.22 work
#### Developer Done List

- [ ] Tests Added/Updated
- [ ] Updated README.md
- [ ] Verified backward compatible
- [ ] Verified database migrations will not be catastrophic
- [ ] Considered Security, Availability and Confidentiality

#### Reviewer/Approval Done List

- [ ] Tests Pass Locally
- [ ] CI Build Passes
- [ ] Verified README.md is updated
- [ ] Verified changes are backward compatible
- [ ] Reviewed impact to Security, Availability and Confidentiality (if issue found, add comments and request changes)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -117,6 +120,6 @@ test-e2e-1.19: container-build install-tools
 test-e2e-1.18: container-build install-tools
 	$(call TEST_KUBERNETES,v1.18.0,$(PREFIX),$(VERSION))
 
-test-e2e-all: test-e2e-1.21.1 test-e2e-1.20 test-e2e-1.19 test-e2e-1.18
+test-e2e-all: test-e2e-1.22 test-e2e-1.21.1 test-e2e-1.20 test-e2e-1.19 test-e2e-1.18
```
Comment (bbylovas) [inline Makefile:124]: Mind removing test-e2e-1.18?

**Review by bbylovas — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (daniel-spray) [reply Makefile:124]: For this 1.22 upgrade we are just removing 1.17. I removed it in a different PR that is why it looks like none got removed.

Comment (bbylovas) [reply Makefile:124]: 👍

**Review by bbylovas — APPROVED** [review]
LGTM

**Pull request merged**